neuralprophet/time_dataset.py

```diff
@@ -455,22 +455,29 @@ def make_country_specific_holidays_df(year_list, country):
     ----------
         year_list : list
             List of years
-        country : string
-            Country name
+        country : str, list
+            List of country names
 
     Returns
     -------
         pd.DataFrame
             Containing country specific holidays df with columns 'ds' and 'holiday'
     """
-
-    try:
-        country_specific_holidays = getattr(hdays_part2, country)(years=year_list)
-    except AttributeError:
+    # iterate over countries and get holidays for each country
+    # convert to list if not already
+    if isinstance(country, str):
+        country = [country]
+    country_specific_holidays = {}
+    for single_country in country:
         try:
-            country_specific_holidays = getattr(hdays_part1, country)(years=year_list)
+            single_country_specific_holidays = getattr(hdays_part2, single_country)(years=year_list)
         except AttributeError:
-            raise AttributeError(f"Holidays in {country} are not currently supported!")
+            try:
+                single_country_specific_holidays = getattr(hdays_part1, single_country)(years=year_list)
+            except AttributeError:
+                raise AttributeError(f"Holidays in {single_country} are not currently supported!")
```

Comment:
Might be good to extract this code (as we have it multiple times) into a helper function? Also happy to have this as a follow up issue.

Reply:
Good point! I think it would make sense to do a follow up issue and consider that together with issue #712 as there might be synergies effects

Reply:
👍